pulsar/producer_partition.go

```diff
@@ -690,10 +693,9 @@ func (p *partitionProducer) internalSend(request *sendRequest) {
 				return
 			}
 		}
-		if request.flushImmediately {
 
+		if request.flushImmediately {
```

Comment:
Try to avoid changing the various formats.

Reply:
I haven't changed the format here, just delete the redunpdant blank lines. If you meant I renamed `internalcrypto` to `internalCrypto` just because my IDE told me that is a typo error.

Reply:
> I haven't changed the format here, just delete the redunpdant blank lines.

I mean, we don't need to change this(Even if it can pass CI). 

>  If you meant I renamed internalcrypto to internalCrypto just because my IDE told me that is a typo error.
I agree to fix the misspelling

Reply:
OK, I have reverted it.